scalableminds/webknossos
# Relax bounding box requirements for model training

Remove requirements of bounding boxes to have the same x/y dimension as well as the same dimensions overall for model training. Needs https://github.com/scalableminds/voxelytics/pull/3796.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Create an annotation with two differently sized bounding boxes.
- Create two annotations each with one bounding box that are differently sized.
- Locally execute `yarn enable-jobs`, open "AI Models" from the Administration pane. Trying to start a training using the one annotation as well as using the two annotations with differently sized bounding boxes should be allowed.

------
(Please delete unneeded items, merge only when none are left open)
